doc/developers.rst

```diff
@@ -18,13 +18,23 @@ with more coverage.
 
 Building Documentation
 ----------------------
+The following is needed in order to build the documentation.
+
+*    ``pip install sphinx``
```

Comment:
you can put all the installers in the same line, no? something like:
```sh
pip install sphinx sphinx-gallery pillow
```
does that work?